### DISCUSSION TITLE
Mapping between the EMS addresses and the robot associated parts

### DISCUSSION THREAD

**Discussion by DanielePucci** [robotology/community#111] [Q&A] [answered]
Is there a documentation where it is explained the mapping between the EMSs and the robot associated parts? For instance, the EMS 6 is in charge of handling the motion control and the strain for the left upper leg. Can we have a complete description for the entire robot?

@marcoaccame @valegagge @ale-git @randaz81

**Comment by pattacini** [comment]
@marcoaccame are you willing to take on this?

**Comment by marcoaccame** [comment]
Hi @pattacini, I can prepare the document. 

Where do you think it is best to publish it?

**Comment by pattacini** [comment]
Thanks @marcoaccame 

Ideally, the place should be our [wiki](http://wiki.icub.org/wiki/Main_Page), but it's better to talk to @randaz81 for that.

**Comment by DanielePucci** [comment]
👍

**Comment by pattacini** [comment]
Hi @marcoaccame any update on this?
If the documentation is ready we could post a link to it here and close the issue.

**Comment by pattacini** [comment]
Ping @marcoaccame

**Comment by traversaro** [comment]
cc @fjandrad

**Comment by marcoaccame** [comment, marked as answer]
The requested mapping table for `ETH` robots is now available in the wiki at: http://wiki.icub.org/wiki/Firmware#ETH_robots.
